core/libdeno/internal.h

```diff
@@ -25,6 +25,56 @@ struct ModuleInfo {
   }
 };
 
+class DenoArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
+ public:
+  ~DenoArrayBufferAllocator() {
+    // TODO(piscisaureus): Enable this check. At the time of writing, it fails
+    // for some tests, indicating that we're leaking buffers.
+    // CHECK(ref_count_map_.empty());
+  }
+
+  void* Allocate(size_t length) override { return new uint8_t[length](); }
```

Comment:
does this need to be zero'd out?

Reply:
Yes the non-initializing flavor is two lines below. 👀